create draft identity errors and metrics

joeniquette opened the pull request:
draft

joeniquette committed:
create draft identity errors and metrics

draft

joeniquette committed:
Update Identity_errors_metrics.md

AGarcia-Clarity commented on teams/vsp/teams/identity/Identity_errors_metrics.md:
```diff
@@ -0,0 +1,29 @@
+The VSP Identity team maintains metrics and errors for all calls related to user data within VA.gov. Below is a list of the locations these metrics and errors can be found in their final presentation state and source code.
+
+
+[Prometheus](http://prometheus-prod.vfs.va.gov:9090/prometheus/alerts) is the main location of our performance alerts. Many of the metrics the Identity team maintains are graphed here and then used for alreting to slack and pagerduty.
```
Alerting*

asg5704 reviewed (commented):
I would also change any reference of Identity team and change it to VSP Identity team to keep consistency.

joeniquette committed:
Update Identity_errors_metrics.md

joeniquette committed:
Update Identity_errors_metrics.md

joeniquette committed:
update metrics and alert links

Still need to expand upon definitions of each one as well as find how `api_auth_login_callback_failed` is generated.
